.chloggen/add-peer-service.yaml

```
@@ -0,0 +1,37 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: peer, aws, azure gcp
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Renames `peer.service` to `peer.service.name` for consistency with `service.name` pattern, and consolidates cloud provider attributes into the peer namespace.
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+# The values here must be integers.
+issues: [2338]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  The following attributes have been renamed:
+
+  Core peer attributes:
+  - `peer.service` → `peer.service.name` (aligns with `service.name` pattern)
+
+  New peer attributes added:
+  - `peer.service.namespace`
+  - `peer.service.id`
+
+  Azure attributes:
+  - `azure.resource_provider.namespace` → `peer.service.namespace`
+  - `az.namespace` → `peer.service.namespace`
+
+  GCP attributes:
+  - `gcp.client.service` → `peer.service.name`
```

```suggestion
  - `gcp.client.service` → `peer.service.namespace`
```

Per https://github.com/open-telemetry/semantic-conventions/pull/2405#discussion_r2265542586.

---

Thanks for spoting that